testrtc.py

```diff
@@ -21,7 +27,45 @@ def get(self, size_kbytes):
     self.response.headers.add_header("Access-Control-Allow-Origin", "*")
     self.response.headers['Content-Type'] = 'application/octet-stream'
     self.response.out.write(random_file[0: int(size_kbytes)*1000])
-    
+
+class NewReportHandler(blobstore_handlers.BlobstoreUploadHandler):
+  # Generate and return blobstore upload URL.
+  def head(self):
+    referer = self.request.headers['Referer']
+    upload_url = blobstore.create_upload_url('/report/new')
+    self.response.headers.add_header("Access-Control-Allow-Origin", referer)
```

Comment:
This header is wrong. E.g. if I visit this from "mydomain.com" it will say allow from "mydomain.com", etc...
Here you should want to allows return 'https://test.webrtc.org/' or so independently where the user is coming from.